Adafruit_GFX.cpp

```diff
@@ -71,596 +71,680 @@ POSSIBILITY OF SUCH DAMAGE.
 #endif
 
 Adafruit_GFX::Adafruit_GFX(int16_t w, int16_t h):
-  WIDTH(w), HEIGHT(h)
```

Comment:
Ah, okay, as long as it is moving in the direction of more consistent than less consistent, great!